cfe_internal/core/watchdog/templates/watchdog-windows.ps1.mustache

```diff
@@ -0,0 +1,23 @@
+$LOGFILE="{{{logfile}}}"
+
+$long_running_cf_agent_threshold_min = 5
+$long_running_cf_agent_count = @(Get-Process cf-agent -erroraction "silentlycontinue" | Where { $_.StartTime -lt (Get-Date).AddMinutes(-$long_running_cf_agent_threshold_min) }).count
+$long_running_cf_agent_count_threshold = 1
+$running_cf_agent_count = @(Get-Process cf-agent -erroraction "silentlycontinue").count
+
+If ($long_running_cf_agent_count -ge $long_running_cf_agent_count_threshold) {
+    $DATESTAMP=Get-Date -Format "yyyy-MM-dd HH:mm"
+
+    Write-Output "${DATESTAMP}: Count of long running cf-agent ($long_running_cf_agent_count) exceeds threshold ($long_running_cf_agent_count_threshold) of long running agents, remediation triggered." | Tee-Object -FilePath "$LOGFILE" -Append
```

Comment:
niptick: I'm not sure about *exact* meaning of word "threshold" in English (no sarcasm here), but note that here you say "exceeds threshold", while in reality it will as well print when number of long running cf-agents equals to threshold. I'd rewrite it like this:

> Count of long running cf-agent **processes** _N_ ~exceeds~ **hits** threshold _M_ ~of long running agents~, remediation triggered

Reply:
Yes, thanks I was previously at -gt 0.